lib/lrmd/lrmd_client.c

```diff
@@ -301,7 +301,8 @@ lrmd_dispatch_internal(lrmd_t * lrmd, xmlNode * msg)
         crm_element_value_epoch(msg, PCMK__XA_LRMD_RCCHANGE_TIME, &epoch);
         event.t_rcchange = (unsigned int) epoch;
 
-        crm_element_value_int(msg, F_LRMD_RSC_EXEC_TIME, (int *)&event.exec_time);
+        crm_element_value_int(msg, PCMK__XA_LRMD_EXEC_TIME,
+                              (int *) &event.exec_time);
```

Comment:
FYI as an aside, this is always a bad idea, though likely fine in this case (unsigned int * -> int *). It's better to create a new variable of the appropriate type to parse, then assign it to the destination variable. (Or create a new helper for the appropriate type.)